code/modules/mob/living/simple_animal/hostile/hostile.dm

```diff
@@ -573,3 +588,61 @@ mob/living/simple_animal/hostile/proc/DestroySurroundings() // for use with mega
 				. += M
 			else if (M.loc.type in hostile_machines)
 				. += M.loc
+
+/**
+  * Proc that handles a charge attack windup for a mob.
+  */
+/mob/living/simple_animal/hostile/proc/enter_charge(var/atom/target)
+	if((mobility_flags & (MOBILITY_MOVE | MOBILITY_STAND)) != (MOBILITY_MOVE | MOBILITY_STAND) || charge_state)
```

Comment:
:thinking:

What's going on here. It's doing an equality check against (already checked) mobility bit flags?